src/codeedges.jl

```diff
@@ -228,6 +231,10 @@ function direct_links!(cl::CodeLinks, src::CodeInfo)
                 end
                 target = P(name, targetstore)
                 add_links!(target, stmt, cl)
+            elseif name in (nothing, false)
+            else
+                @show stmt
+                error("name ", typeof(name), " not recognized")
```

Comment:
Is this transient or permanent? I'm fine with it either way, just curious.

Reply:
Should be unreachable in current julia, but intended to fail loudly if julia base changes.